<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_AddUser</class>
 <widget class="QFrame" name="F_AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="L_header">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMain">
     <item>
      <layout class="QVBoxLayout" name="layoutLeftSide">
       <item>
        <widget class="QLabel" name="L_firstName">
         <property name="text">
          <string>First Name *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_FirstName">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_lastName">
         <property name="text">
          <string>Last Name *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_lastName">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_gender">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_gender">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_company">
         <property name="text">
          <string>Company *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_company">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_phone">
         <property name="text">
          <string>Phone *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_phone">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_email">
         <property name="text">
          <string>Email </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_email">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_username">
         <property name="text">
          <string>Username *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_username">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_password">
         <property name="text">
          <string>Password *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_password">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_info">
         <property name="text">
          <string>At least 1 Capital, 1 Lowercase, 1 number and more 
 then 8 characters long</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_confirmePass">
         <property name="text">
          <string>Confirm Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_confirmePass">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutRightSide">
       <item>
        <widget class="QLabel" name="L_status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_status">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Active</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Banned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Disactive</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delete</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_group">
         <property name="text">
          <string>Group *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_group"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
